ch04-git-on-the-server.asc

```diff
@@ -38,11 +37,11 @@ include::book/04-git-server/sections/gitlab.asc[]
 
 include::book/04-git-server/sections/hosted.asc[]
 
-=== Summary
+=== özet
 
-You have several options to get a remote Git repository up and running so that you can collaborate with others or share your work.
+Başkalarıyla işbirliği yapabilmeniz veya çalışmalarınızı paylaşabilmeniz ve uzak bir Git deposunu çalıştırmak için birkaç seçeneğiniz vardır
```

Comment:
Cümle nokta (`.`) ile bitiyor normalde; fakat burdan sonrası bu cümleye bağlı açıklamalar olduğu için iki nokta (`:`) ile de bitebilir. Seçimi size bırakıyorum.